src/psyclone/transformations.py

```diff
@@ -2262,3 +2262,82 @@ def apply(self, sched):
         schedule.addchild(data_dir, index=0)
 
         return schedule, keep
+
+
+class ACCRoutineTrans(Transformation):
+    '''
+    Transform a kernel subroutine through the addition of a "!$acc routine"
+    directive (causing it to be compiled for the OpenACC accelerator device).
+    For example:
+
+    >>> from psyclone.parse import parse
+    >>> from psyclone.psyGen import PSyFactory
+    >>> api = "gocean1.0"
+    >>> filename = "nemolite2d_alg.f90"
+    >>> ast, invokeInfo = parse(filename, api=api, invoke_name="invoke")
+    >>> psy = PSyFactory(api).create(invokeInfo)
+    >>>
+    >>> from psyclone.transformations import ACCRoutineTrans
```

Comment:
Just a thought ... it might be better to separate transformations on kernel, psy and algorithm layers in the future ... e.g. this could then become "from psyclone.transformations.kernel import ACCRoutineTrans"

Reply:
Good point. I've created #211 for this.

Reply:
Thanks